policy-controller/k8s/index/src/index.rs

```diff
@@ -1474,3 +1480,65 @@ impl<T> Default for NsUpdate<T> {
         }
     }
 }
+
+// === impl ClusterInfo ===
+
+impl ClusterInfo {
+    fn default_inbound_http_routes<'p>(
+        &self,
+        probe_paths: impl Iterator<Item = &'p str>,
+    ) -> HashMap<InboundHttpRouteRef, InboundHttpRoute> {
+        let mut routes = HashMap::with_capacity(2);
+        // If no routes are defined for the server, use a default route that
+        // matches all requests. Default authorizations are instrumented on
+        // the server.
+        routes.insert(
+            InboundHttpRouteRef::Default("default"),
+            InboundHttpRoute::default(),
+        );
+
+        // No Server is configured, so requests on the Pod's probe paths are
+        // authorized.
+        if !self.probe_networks.is_empty() {
+            // Probes are authorized on the configured probe networks only.
+            let authorizations = Some((
+                AuthorizationRef::Default("probe"),
+                ClientAuthorization {
+                    networks: self
+                        .probe_networks
+                        .iter()
+                        .copied()
+                        .map(Into::into)
+                        .collect(),
+                    authentication: ClientAuthentication::Unauthenticated,
+                },
+            ))
+            .into_iter()
```

Comment:
nit/TIOLI: alternatively, instead of `Option::into_iter`, we could also use [`std::iter::once`](https://doc.rust-lang.org/std/iter/fn.once.html) here, which maaaaaybe expresses the intent a little more clearly? not a big deal either way though.
```suggestion
            let authorizations = std::iter::once((
                AuthorizationRef::Default("probe"),
                ClientAuthorization {
                    networks: self
                        .probe_networks
                        .iter()
                        .copied()
                        .map(Into::into)
                        .collect(),
                    authentication: ClientAuthentication::Unauthenticated,
                },
            ))
```